tests/php/sync/test_class.jetpack-sync-options.php

```diff
@@ -179,14 +179,18 @@ public function test_sync_default_options() {
 			'uploads_use_yearmonth_folders'        => '0',
 			'date_format'                          => '0',
 			'time_format'                          => '0',
+			'mailserver_url'                       => 'pineapple.example.com',
+			'mailserver_login'                     => '',
+			'mailserver_pass'                      => '',
```

Comment:
If I would add values like "pineapple" for `mailserver_login` and `mailserver_pass `, I would need to fetch the list of contentless options and create an exception for any options on that list when their values are tested in:

```php
foreach ( $options as $option_name => $value ) {
  $this->assertOptionIsSynced( $option_name, $value );
}